<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bouton_employe</class>
 <widget class="QDialog" name="bouton_employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>departement employ√©</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>employe</string>
    </attribute>
    <widget class="QPushButton" name="ajoutereButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>544</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifemployeButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>544</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ln">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_ln">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu_de_naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>date_de_naissance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_supprimere">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_nom">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_nationalite">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_dn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>251</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_prenom">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nationalite">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifier_cin">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>230</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu_de_naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>nationalite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nationalite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>200</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>date_de_naissance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimereButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>544</x>
       <y>170</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cin</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>metiers</string>
    </attribute>
    <widget class="QLineEdit" name="mail_pass">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>your mail password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="placeholderText">
      <string>attachement here</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="placeholderText">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rechercher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="placeholderText">
      <string>recipient</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdf_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>270</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rcin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>rechercher cin</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
     <property name="placeholderText">
      <string>type your message here ...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rcin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rprenom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>rechercher prenom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>Browse ...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recherchernom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 241, 241)</string>
     </property>
     <property name="text">
      <string>rechercher nom</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="recuButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>270</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>g√©n√©rer un re√ßu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_n">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rcin_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rcin_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_s">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>    CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>110</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>salaire</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
